Bottom: a68ab8a917668a647e0e91bdcdfa709068548aee
Top:    4c20a9f28f13787ad0f47477c9dc613dfb0f6080
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-21 13:04:11 +0000

parse_l.l: Fix warning implicit definition of function ‘CreateBeLenient’

Warning was introduced in commit e8dd18a2f7b83c6108d2c9398903d4be92e22d4c
The required prototype was added to create.h, but create.h was not
included in parse_l.l

Note to developers:

Please "make clean" and build with "make -s" before pushing changes.
This kind of bug is really easy to catch, and I just recently spent
a log of effort cleaning up PCB's compile so we can catch real bugs!

Pretty please?


---

diff --git a/src/parse_l.l b/src/parse_l.l
index 4ed5741..9799250 100644
--- a/src/parse_l.l
+++ b/src/parse_l.l
@@ -61,6 +61,7 @@ RCSID("$Id$");
 #include "strflags.h"
 #include "parse_l.h"
 #include "parse_y.h"
+#include "create.h"
 
 #define YY_NO_INPUT
